app.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express")
+const cors = require("cors")
+const app = express();
+const bodyParser = require("body-parser")
+const port =3000;
+const school = require("./School.js")
+app.use(cors())
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({extended:false}))
+
+let newSchool = new school ()
+// console.log(newSchool)
+
+app.get("/school",(req,res)=>{
+    res.json(newSchool.classes)
+})
+
+app.post("/school/add/class",(req,res)=>{
```

Comment:
this endpoint should look like `/class` and have `req.body`. This was specified on the readme file